core/src/main/java/org/kohsuke/stapler/StaplerRequest.java

```diff
@@ -497,6 +498,7 @@ public interface StaplerRequest extends HttpServletRequest {
      *      This includes the case where the name corresponds to a simple
      *      form field (like textbox, checkbox, etc.) 
      */
+    @WithBridgeMethods(org.apache.commons.fileupload.FileItem.class)
```

Comment:
Bridge methods need to be added to both interface and implementation. See the unit tests for `bridge-method-injector`.